frontend/src/components/Logo/LogoFooter.jsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
+import { HashLink } from 'react-router-hash-link';
 
 const LogoFooter = () => (
   <div className="fr-footer__brand">
-    <a className="fr-logo" href="/" title="République française">
+    <HashLink className="fr-logo" to="/" title="République française">
```

Comment:
Shouldn't this point to MESRI website?

Reply:
i'll ask in the chat and create a ticket if needed